ui/src/app/pages/access/access-renewal-page.tsx

```diff
@@ -263,9 +364,10 @@ export const AccessRenewalPage = fp.flow(
         <ActionButton isComplete={!isExpiring(getExpirationTimeFor('dataUseAgreement'))}
           actionButtonText='View & Sign'
           completedButtonText='Completed'
-          onClick={() => navigate(['data-code-of-conduct'])}
+          onClick={() => navigateByUrl('data-code-of-conduct?renewal=1')}
```

Comment:
Use a query param to let the page know it was referred by renewal. Using history would introduce a lot of complexity